### PR TITLE
docs(button-toggle-group): update story to use csf

### PR DESCRIPTION
**Pull request by edleeks87**
### Proposed behaviour

Moves story to `Test` section, updates to use `csf` and removes initial value from story to allow easier
cypress testing

### Current behaviour

There is an initial value set in the story and it is written using `storiesOf`

### Checklist

- [x] Release notes (Conventional Commits) 
<del>- [ ] Unit tests<del>
- [ ] Cypress automation tests
- [x] Storybook added or updated
<del>- [ ] Theme support<del>
<del>- [ ] Typescript `d.ts` file added or updated<del>

### CODE REVIEW

File: src/components/button-toggle-group/button-toggle-group.stories.js [hunk 1]
```diff
@@ -1,13 +1,11 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import {
   text, number, boolean, select, percentageRange
 } from '@storybook/addon-knobs';
 import { action } from '@storybook/addon-actions';
 import { State, Store } from '@sambego/storybook-state';
 import { dlsThemeSelector, classicThemeSelector } from '../../../.storybook/theme-selectors';
 import OptionsHelper from '../../utils/helpers/options-helper';
-import notes from './documentation';
```
Comment (mkrds) [inline src/components/button-toggle-group/button-toggle-group.stories.js]: Shouldnt this file also be removed?

Comment (edleeks87) [reply src/components/button-toggle-group/button-toggle-group.stories.js]: I left it as there is another ticket to add the `mdx` file in and some of the content seemed relevant to add there

Comment (mkrds) [reply src/components/button-toggle-group/button-toggle-group.stories.js]: Makes sense :)

**Pull request merged**